tests/SchemaTest.php

```diff
@@ -573,23 +585,23 @@ public function testVectorSearchIndex()
         self::assertSame('vector', $index['latestDefinition']['fields'][0]['type']);
     }
 
-    protected function getIndex(string $collection, string $name)
+    protected function getIndex(string $collection, string $name): ?IndexInfo
```

Comment:
Should I cast `IndexInfo` to an array for future compatibility with mongodb/mongodb v2.0?